src/views/start.njk

```diff
@@ -49,26 +49,44 @@
         }) }}
       </form>
       <p class="govuk-body">
-      If you need assistance with this service, contact the NatureScot Licensing Team.
+        If you need assistance with this application, contact the NatureScot
```

Comment:
did this need to change?

Reply:
nah this is my bad, ill fix

Reply:
Reverted the text that i changed by mistake

Reply:
katherine would like this text to be - 'If you need assistance with this service, contact the NatureScot Licensing Team'